classes/site/SiteDAO.inc.php

```diff
@@ -18,6 +18,12 @@
 
 class SiteDAO extends DAO {
 
+	/** @var string One of the SCHEMA_... constants */
+	var $schemaName = 'site';
```

Comment:
This should still be `= SCHEMA_SITE` rather than re-introducing the string constant, I think.

Reply:
There was a problem associated with it, I don't remember exactly. Does PHP allow that?

Reply:
Yes, you can use a constant to initialize a class member (if that's the problem you're thinking of) -- all I can think is that maybe you were using the constant before it was defined. But anyway, I think this takes precedence: https://github.com/pkp/ojs/pull/2873#discussion_r493912372